readme.md

```diff
@@ -1,19 +1,37 @@
 # GitHub Actions Kubernetes
 
-A collection of GitHub actions for Kubernetes projects.
+A collection of GitHub actions for Kubernetes.
 
-## Example Playbook
+## Setup DigitalOcean
+
+Setup Digital-Ocean CLI and configure Kubernetes.
+
+### Example
+
+```yaml
+  - uses: aboutbits/github-actions-kubernetes/setup-kubectl@v1
```

Comment:
oh, that is what you mean by `do`. I think that is very confusing considering the english word for `machen`.

Reply:
how about `setup-digital-ocean-kubectl`? @alexlanz

Reply:
Yes, it could be confusing first. But we did the same also in our infrastructure project: https://github.com/aboutbits/infrastructure/tree/main/terraform/modules

I would suggest to prefix the actions with the vendor name. Otherwise, for other examples it could be difficult to come up with a good name that also works. Here it is working, but it is not working always. Plus, they are really nice grouped in the folder structure.

I chose `do` because also for others normally short names are used: aws, gcp.